packages/flutter/lib/src/material/progress_indicator.dart

```diff
@@ -593,7 +593,9 @@ class _CircularProgressIndicatorState extends State<CircularProgressIndicator> w
   }
 
   Widget _buildCupertinoIndicator(BuildContext context) {
-    return CupertinoActivityIndicator(key: widget.key);
+    final Color? tickColor = widget.backgroundColor ?? ProgressIndicatorTheme.of(context).circularTrackColor;
```

Comment:
Delegating to Material's ProgressIndicatorTheme here is a backwards incompatible change and probably not what a Cupertino app would expect. It may also be why there were some golden image failures.